docs/7.x/changelog.md

```diff
@@ -1,42 +1,44 @@
 # Releases
 
-## Supported Branches
+## Supported Versions
 
-| Branch | LTS | Latest Release | Latest Release Date | Initial Release Date | End of Support          | Kubernetes Version   | Teleport Version |
-| ------ | --- | -------------- | ------------------- | -------------------- | ----------------------- | -------------------- | ---------------- |
-| 7.0    | No  | 7.0.9          | June 16, 2020       | April 3, 2020        | 7.1 is released         | 1.17.6               | 3.2.13           |
-| 6.1    | Yes | 6.1.29         | June 12, 2020       | August 2, 2019       | November 10, 2021       | 1.15.12              | 3.2.12           |
-| 5.5    | Yes | 5.5.48         | June 15, 2020       | March 8, 2019        | March 8, 2021           | 1.13.11              | 3.0.6-gravity    |
+Find the latest releases of supported Gravity versions at [Gravity Downloads](https://gravitational.com/gravity/download).
```

Comment:
I wonder if we should specifically indicate this as Gravity OSS.

Reply:
Probably.  If we indicate this is OSS only, I'd like to give a pointer to enterprise users too.  However I haven't found the a similar resource to the OSS downloads page.  Helgi's question leads me to believe it may be this changelog page.

Reply:
No, as I mentioned in slack, the references to enterprise downloads got completely removed from the docs for some reason. As such, currently someone from professional services needs to instruct how to get the enterprise downloads.

Reply:
Specified OSS in 5f84e8a.